test/Microsoft.TestPlatform.AdapterUtilities.UnitTests/TestIdProvider/SHA1ImplTests.cs

```diff
@@ -138,7 +136,7 @@ private void SHA1_TestRepetitionVector(char input, int repetition, string expect
         }
         else
         {
-            expected = expected.Replace(" ", "").ToLowerInvariant();
+            expected = expected!.Replace(" ", "").ToLowerInvariant();
```

Comment:
In net48, `string.IsNullOrEmpty` doesn't provide the non-null info to the analyzer. When we will use our shimlayer, we could have our own implementation that would provide this hint to the compiler. In the meantime, I went with bang.